docs/onboarding.md

```diff
@@ -0,0 +1,39 @@
+# How to build your RPM(s) in Konflux
```

Comment:
Should there be a note (a step) about OCI (Quay.io)?

Reply:
There is, `Download the RPMs from Quay.io`.